driver/apdu/gbinder_hidl.c

```diff
@@ -27,19 +27,22 @@
 
 static int lastChannelId = -1;
 
-struct radio_response_info {
+struct radio_response_info
+{
```

Comment:
why change code style?

Reply:
My editor formatted it automatically.

Reply:
Looks like we need a .clang-format config.